doc/user/content/sql/system-catalog/mz_internal.md

```diff
@@ -1003,3 +1003,4 @@ The `mz_scheduling_parks_histogram` view describes a histogram of [dataflow] wor
 <!-- RELATION_SPEC_UNDOCUMENTED mz_internal.mz_storage_usage_by_shard -->
 <!-- RELATION_SPEC_UNDOCUMENTED mz_internal.mz_view_foreign_keys -->
 <!-- RELATION_SPEC_UNDOCUMENTED mz_internal.mz_view_keys -->
+<!-- RELATION_SPEC_UNDOCUMENTED mz_internal.mz_object_lifetimes -->
```

Comment:
Curious why this is undocumented?

Reply:
I was just lazy on the first iteration 😊 added documentation now though!